codebuild/cd/fips-jar-build.yml

```diff
@@ -0,0 +1,31 @@
+version: 0.2
+#this buildspec assumes the aws-common-runtime/ubuntu-16.04 image
+# This job is responsible for artifacting the JAR which will have all of the other shared libs stuffed
+# into it once all platforms are built and artifacted
+phases:
+  install:
+    commands:
+      - sudo add-apt-repository ppa:openjdk-r/ppa
+      - sudo apt-get update -y
+      - sudo apt-get install openjdk-11-jdk-headless maven wget unzip -y -f
+  build:
+    commands:
+      - cd $CODEBUILD_SRC_DIR/aws-crt-java
+      # upload artifacts to S3
+      - export GIT_TAG=$(git describe --tags)
+      - mkdir -p target/cmake-build/lib
+      # prepare FIPS uber jar, download the regular libs
+      - aws s3 cp --recursive s3://aws-crt-java-pipeline/${GIT_TAG}/lib $CODEBUILD_SRC_DIR/aws-crt-java/target/cmake-build/lib
+      # Override with the FIPS libs
+      - aws s3 cp --recursive s3://aws-crt-java-pipeline/${GIT_TAG}/fips_lib $CODEBUILD_SRC_DIR/aws-crt-java/target/cmake-build/lib
```

Comment:
again, spooked about accidentally uploading fips jar without the expected fips libs. Can we assert that `s3://.../fips_lib` has the expected files in it?

Reply:
well, the fips shared lib will have the same structure and name as the regular one, I don't think we can assert on the files to make sure it works.

We can probably run a test instead.